wannierberri/__sym_wann.py

```diff
@@ -91,197 +111,210 @@ def __init__(self,num_wann=None,lattice=None,positions=None,atom_name=None,proj=
+            dmat[0, 0] = np.exp(-(alpha + gamma) / 2.0 * 1j) * np.cos(beta / 2.0)
+            dmat[0, 1] = -np.exp(-(alpha - gamma) / 2.0 * 1j) * np.sin(beta / 2.0)
+            dmat[1, 0] = np.exp((alpha - gamma) / 2.0 * 1j) * np.sin(beta / 2.0)
+            dmat[1, 1] = np.exp((alpha + gamma) / 2.0 * 1j) * np.cos(beta / 2.0)
+        rot_orbital = self.orbitals.rot_orb(orb_symbol, rot_sym_glb)
         if self.soc:
-            rot_orbital = np.kron(dmat,rot_orbital)
+            rot_orbital = np.kron(dmat, rot_orbital)
             rot_imag = rot_orbital.imag
             rot_real = rot_orbital.real
-            rot_orbital = np.array(rot_real + 1j*rot_imag,dtype=complex)
+            rot_orbital = np.array(rot_real + 1j * rot_imag, dtype=complex)
         return rot_orbital
 
-    def atom_rot_map(self,sym):
+    def atom_rot_map(self, sym):
         '''
         rot_map: A map to show which atom is the equivalent atom after rotation operation.
         vec_shift_map: Change of R vector after rotation operation.
         '''
         wann_atom_positions = [self.wann_atom_info[i][2] for i in range(self.num_wann_atom)]
-        rot_map=[]
-        vec_shift_map=[]
+        rot_map = []
+        vec_shift_map = []
         for atomran in range(self.num_wann_atom):
-            atom_position=np.array(wann_atom_positions[atomran])
-            new_atom = np.dot(self.symmetry['rotations'][sym],atom_position) + self.symmetry['translations'][sym]
+            atom_position = np.array(wann_atom_positions[atomran])
+            new_atom = np.dot(self.symmetry['rotations'][sym], atom_position) + self.symmetry['translations'][sym]
             for atom_index in range(self.num_wann_atom):
-                old_atom= np.array(wann_atom_positions[atom_index])
-                diff = np.array(new_atom-old_atom)
-                if np.all( abs((diff+0.5)%1-0.5)<1e-5 ):
-                    match_index=atom_index
-                    vec_shift= np.array(np.round(new_atom-np.array(wann_atom_positions[match_index]),decimals=2),dtype=int)
+                old_atom = np.array(wann_atom_positions[atom_index])
+                diff = np.array(new_atom - old_atom)
+                if np.all(abs((diff + 0.5) % 1 - 0.5) < 1e-5):
+                    match_index = atom_index
+                    vec_shift = np.array(
+                        np.round(new_atom - np.array(wann_atom_positions[match_index]), decimals=2), dtype=int)
                 else:
-                    if atom_index==self.num_wann_atom-1:
-                        assert atom_index != 0,'Error!!!!: no atom can match the new one Rvec = {}, atom_index = {}'.format(self.iRvec[ir],atom_index)
+                    if atom_index == self.num_wann_atom - 1:
+                        assert atom_index != 0
+                        # 'Error!!!!: no atom can match the new one Rvec = {}, atom_index = {}'.format(
+                        #     self.iRvec[ir], atom_index)
```

Comment:
```suggestion
                    if atom_index==self.num_wann_atom-1:
                        assert atom_index != 0,(
                            f'Error!!!!: no atom can match the new atom after symmetry operation {sym+1},\n'
                            +f'Before operation: atom {atomran} = {atom_position},\n'
                            +f'After operation: {atom_position},\n'
                            +f'All wann_atom: {wann_atom_positions}'
                            )
```
Could you please make this change to solve the undefined variable in __sym_wann.py